pkg/gitserver/util/hooks.go

```diff
@@ -0,0 +1,30 @@
+package util
```

Comment:
Don't create util packages.  Create a package named for what this package is going to accomplish, like `pkg/gitserver/hooks`.